### ISSUE TITLE
Crash - aws provider nil pointer

### ISSUE THREAD

**Issue by ryanmcnamara** [terraform-providers/terraform-provider-aws#3953]
### Terraform Version
0.10.7

I had a plan fail with this message at the end of STDOUT

```
!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Terraform crashed! This is always indicative of a bug within Terraform.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Terraform[1] so that we can fix this.

When reporting bugs, please include your terraform version. That
information is available on the first line of crash.log. You can also
get it by running 'terraform --version' on the command line.

[1]: https://github.com/hashicorp/terraform/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! TERRAFORM CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Here is an excerpt that contains the nil pointer:
https://gist.github.com/ryanmcnamara/90dfe7f1230e25d5be2be2a4b7c0e572

I have the full output from the plan which I could direct message to someone on behalf of terraform

**Comment by bflad** [comment]
Hi @ryanmcnamara! 👋  Sorry you ran into trouble here. We actually just patched a fix for this into master via #3954 which will release with v1.13.0 of the AWS provider, which is planned to ship next week. Given that I'm going to go ahead and close this issue.

Was your configuration running in sa-east-1 by chance?

**Comment by guodongx** [comment]
I think we hit this issue on us-east-1

**Comment by bflad** [comment]
I can confirm v1.12.0 now crashes in us-east-1 myself but still not us-west-2. My guess is that they are rolling a service update out by region. We might need to push out v1.13.0 a little early. 😅

**Comment by eedgar** [comment]
We have run into this as well.

**Comment by cmvelo** [comment]
Also running into this issue which is severely impeding our infrastructure rollout 👎

**Comment by bflad** [comment]
Please don't be alarmed, but locking this issue to focus everyone to the original report #3949. The fix is in master and we likely are going to release v1.13.0 immediately.

**Comment by bflad** [comment]
This has been released in [version 1.13.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1130-march-28-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
